app/gui/view/graph-editor/src/component/node.rs

```diff
@@ -306,6 +307,7 @@ ensogl::define_endpoints_2! {
         edit_expression       (text::Range<text::Byte>, ImString),
         set_skip_macro        (bool),
         set_freeze_macro      (bool),
+        set_context_switch    (bool),
```

Comment:
Let's add basic documentation for this endpoint, it is not clear what `bool` represents here. Same for graph editor endpoint.